config/log_rules.json

```diff
@@ -125,6 +125,22 @@
         "end_message": "[LOG] %{file}(line %{line}): %{message}",
         "type": "error"
       },
+      "build_pipeline_message": {
+        "active": true,
+        "start_pattern": "UnityEditor\\.BuildPipeline:BuildPlayerInternalNoCheck",
+        "fetch_first_line_not_matching": [
+          "UnityEditor\\.",
+          "^\\n",
+          "^\\t",
+          "^  "
+        ],
+        "fetched_line_pattern": "(?<message>.*)\\n",
+        "fetched_line_message": false,
+        "start_message": false,
+        "end_pattern": "Filename: (?:.+/(?<file>\\w+\\.\\w+\\.\\w+))? Line: (?<line>-?\\d+)",
+        "end_message": "[LOG] %{file}(line %{line}): %{message}",
+        "type": "warning"
```

Comment:
I was wondering if we should make this a warning or not. But I guess yes.

Reply:
I initially wrote it as a simple message, but after testing it on a few log files, I noticed that most of those messages are actually warnings, so I opted for that :)

Reply:
We do need the test suite tho